rl_zoo3/train.py

```diff
@@ -152,6 +153,12 @@ def train():
         default=False,
         help="if toggled, display a progress bar using tqdm and rich",
     )
+    parser.add_argument(
```

Comment:
quick remark (will do full review next week): why not use `nargs="+"`?
so you can do `--tags tag1 tag-2 tag3` and not worry about comma and quotes

Reply:
Indeed, it is better, I did not know this feature.